src/game-loop/CMakeLists.txt

```diff
@@ -333,6 +335,15 @@ add_library(GameLoop STATIC
         include/other/LootCollectedEvent.hpp
         include/other/NpcType.hpp
         include/other/ParticleGenerator.hpp
+
+        include/benchmarking/Scene.hpp
+        include/benchmarking/FlaresScene.hpp
+        include/benchmarking/PropsScene.hpp
+        include/benchmarking/RockMassacreScene.hpp
+)
+
+target_compile_definitions(
+        GameLoop PRIVATE $<IF:$<BOOL:${SPELUNKY_PSP_BENCHMARK_MODE}>,SPELUNKY_PSP_BENCHMARK_MODE,>
```

Comment:
As mentioned in description, still not sure whether to make benchmarking a build option or a commandline argument.